the docs are autogenerated by <binary name> -M, it all boils down to this source
file; I'm removing all the man errors from here

--- a/src/generic_agent.c
+++ b/src/generic_agent.c
@@ -1701,7 +1701,7 @@
    {
    if (options[i].has_arg)
       {
-      printf("--%-12s, -%c value - %s\n",options[i].name,(char)options[i].val,hints[i]);
+      printf("--%-12s, -%c - %s\n",options[i].name,(char)options[i].val,hints[i]);
       }
    else
       {
@@ -1728,12 +1728,12 @@
 printf(".SH SYNOPSIS:\n\n %s [options]\n\n.SH DESCRIPTION:\n\n%s\n",GetArg0(component),id);
 
 printf(".B cfengine\n"
-       "is a self-healing configuration and change management based system. You can think of"
+       "is a self-healing configuration and change management based system. You can think of \n"
        ".B cfengine\n"
        "as a very high level language, much higher level than Perl or shell. A"
        "single statement is called a promise, and compliance can result in many hundreds of files"
        "being created, or the permissions of many hundreds of"
-       "files being set. The idea of "
+       "files being set. The idea of \n"
        ".B cfengine\n"
        "is to create a one or more sets of configuration files which will"
        "classify and describe the setup of every host in a network.\n");
@@ -1744,7 +1744,7 @@
    {
    if (options[i].has_arg)
       {
-      printf(".IP \"--%s, -%c\" value\n%s\n",options[i].name,(char)options[i].val,hints[i]);
+      printf(".IP \"--%s, -%c\"\n%s\n",options[i].name,(char)options[i].val,hints[i]);
       }
    else
       {
@@ -1757,10 +1757,10 @@
        ".SH INFORMATION\n");
 
 printf("\nBug reports: http://bug.cfengine.com, ");
-printf(".pp\nCommunity help: http://forum.cfengine.com\n");
-printf(".pp\nCommunity info: http://www.cfengine.org\n");
-printf(".pp\nSupport services: http://www.cfengine.com\n");
-printf(".pp\nThis software is Copyright (C) 2008- Cfengine AS.\n");
+printf(".PP\nCommunity help: http://forum.cfengine.com\n");
+printf(".PP\nCommunity info: http://www.cfengine.org\n");
+printf(".PP\nSupport services: http://www.cfengine.com\n");
+printf(".PP\nThis software is Copyright (C) 2008- Cfengine AS.\n");
 }
 
 /*******************************************************************/
--- a/src/verify_packages.c
+++ b/src/verify_packages.c
@@ -1911,7 +1911,7 @@
 	  }
        else
 	  {
-          CfOut(cf_verbose, "", " Package sucessfully verified from return code");
+          CfOut(cf_verbose, "", " Package successfully verified from return code");
 	  }
        }
     }
